bamboolean/tests/test_normalize.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from bamboolean.factories import normalize
+from . import fixtures
+
+
+class NormalizeExpr(unittest.TestCase):
+    def test_normalize_should_not_change_normalized_terms(self):
```

Comment:
It's nice to have the testing code be as minimal as is reasonable. E.g. in this case if you inline all the values not only the chance of having a bad test is lower, but the test is easier to read.

What is the gain of using a shared fixture?

Reply:
Yeaaah I guess I've just used them without really thinking about this. Done